### PR TITLE
DOCS-2601: add applying surcharges section to woocommerce user guide

### PR DESCRIPTION
**Pull request by Sarah-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - /integrations/woocommerce/faq/payment-fee-surcharges/
+---
+
+Applying [surcharges](/security-and-legal/payment-regulations/about-surcharges/) or payment fees is no longer supported in the Woocommerce plugin.
+
+**Using third party packages**
+To apply a surcharge or payment fee to a payment method, you can use third-party packages that follow the WooCommerce and Wordpress development guidelines.
+
+**Support**
+The Integration Team will do their best to support you with installing a third-party package, but bear in mind that it is a third-party package. We can't guarantee perfect compatibility.
```
Comment (FrancescaNCook) [inline content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md:17]:
```suggestion
The Integration Team will do their best to support you with installing third-party packages, but we can't guarantee perfect compatibility.
```

File: content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
+layout: "faqdetail"
+read_more: "."
+aliases: 
+    - /integrations/woocommerce/faq/payment-fee-surcharges/
+---
+
+Applying [surcharges](/security-and-legal/payment-regulations/about-surcharges/) or payment fees is no longer supported in the Woocommerce plugin.
+
+**Using third party packages**
```
Comment (FrancescaNCook) [inline content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md:13]:
```suggestion
**Using third-party packages**
```

**Review by FrancescaNCook — APPROVED** [review]

File: content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+title : "Applying surcharges"
+meta_title: "WooCommerce plugin - Applying surcharges  - MultiSafepay Docs"
+meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
+layout: "faqdetail"
+read_more: "."
+aliases: 
+    - /integrations/woocommerce/faq/payment-fee-surcharges/
+---
+
+Applying [surcharges](/security-and-legal/payment-regulations/about-surcharges/) or payment fees is no longer supported in the Woocommerce plugin.
```
Comment (Daniel-MultiSafepay) [inline content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md:11]: WooCommerce plugin never supported fees based on the payment method selected in the past.

Comment (Sarah-MultiSafepay) [reply content/payments/integrations/ecommerce-platforms/woocommerce/faq/applying-surcharges.md:11]: Thanks @Daniel-MultiSafepay! I've made a proposed change in this PR: https://github.com/MultiSafepay/docs/pull/1618 can you review it?

**Pull request merged**